src/_datebox.scss

```diff
@@ -0,0 +1,17 @@
+.datebox {
+  display: grid;
+  background-color: $color--primary;
+  grid-area: 3/1/4/5;
+}
+
+@media (min-width: 481px) {
+  .datebox {
+    grid-area: 2/1/3/3;
+    }
```

Comment:
Check code formatting

Reply:
Unfortunately, I don't understand what do You mean.

Reply:
```
.datebox {
   grid-area: 2/1/3/3;
   }
^^
```
There are two unneeded spaces here. Should be:
```
.datebox {
   grid-area: 2/1/3/3;
 }
```
Check other files for this issue too.